### PR TITLE
feat(pandas/dask/polars): implement ops.Pi, ops.E

### PR DESCRIPTION
**Pull request by krzysztof-kwitt**

### CODE REVIEW

File: ibis/backends/polars/compiler.py [hunk 1]
```diff
@@ -978,3 +978,13 @@ def elementwise_udf(op):
     return_type = to_polars_type(op.return_type)
 
     return pl.map(func_args, lambda args: op.func(*args), return_dtype=return_type)
+
+
+@translate.register(ops.E)
+def execute_e(op, **kwargs):
+    return [np.e]
```
Comment (cpcloud) [inline ibis/backends/polars/compiler.py:985]:
```suggestion
    return pl.lit(np.e)
```

File: ibis/backends/polars/compiler.py [hunk 2]
```diff
@@ -978,3 +978,13 @@ def elementwise_udf(op):
...
+
+
+@translate.register(ops.Pi)
+def execute_pi(op, **kwargs):
+    return [np.pi]
```
Comment (cpcloud) [inline ibis/backends/polars/compiler.py:990]:
```suggestion
    return pl.lit(np.pi)
```

**Review by cpcloud — CHANGES_REQUESTED** [review]
LGTM! I think we should `pl.lit` instead of lists though. After that, this is ready to merge.

**Comment by krzysztof-kwitt** [comment]
@cpcloud Makes sense. Updated.

**Review by cpcloud — APPROVED** [review]
Merging!

**Pull request merged**